src/element_ref/element.rs

```diff
@@ -115,4 +115,81 @@ impl<'a> Element for ElementRef<'a> {
         }
```

Comment:
here I've added a couple of tests to validate my patch. I could not do that in the documentation (up in this file) because I needed to import `use html5ever::LocalName;` and that did work.

Not sure on how I can improve this and move all the tests up.